src/Cli/dotnet/commands/dotnet-test/VSTestFeatureFlag.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.DotNet.Tools.Test
+{
+    // !!! FEATURES MUST BE KEPT IN SYNC WITH https://github.com/microsoft/vstest/blob/main/src/Microsoft.TestPlatform.CoreUtilities/FeatureFlag/FeatureFlag.cs !!!
+    internal class FeatureFlag
+    {
+        private const string Prefix = "VSTEST_FEATURE_";
+
+        public Dictionary<string, bool> FeatureFlagsBag { get; } = new();
```

Comment:
This is named `FeatureFlags` in the other class, I think that's better than bag.